include/sort_inference.h

```diff
@@ -0,0 +1,187 @@
+/*********************                                                        */
+/*! \file sort_inference.h
+** \verbatim
+** Top contributors (to current version):
+**   Makai Mann
+** This file is part of the smt-switch project.
+** Copyright (c) 2020 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief Utility functions for checking sortedness and computing the
+**        expected sort when building a term.
+**
+**/
+
+#pragma once
```

Comment:
it is not consistent with the rest of the code.
(even though I like the use of pragma once)

Reply:
Good point -- made the change! We could change to pragma once in the future.